cmd/stat.go

```diff
@@ -209,11 +210,22 @@ func createStatOutputFile(input string) *os.File {
 // createOutputFileWithSuffix creates an output file named after prefixPath
 // appended with suffix.
 func createOutputFileWithSuffix(prefixPath, suffix string) *os.File {
-	output, err := os.Create(prefixPath + suffix)
+	fname := prefixPath + suffix
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		die("failed to get hostname: %s", err)
+	}
+
+	output, err := os.Create(fmt.Sprintf("%s%s.%s.%d", prefixPath, suffix, hostname, os.Getpid()))
 	if err != nil {
 		die("failed to create output file: %s", err)
 	}
 
+	if err = os.Symlink(output.Name(), fname); err != nil {
+		die("failed to create symlink: %s", err)
```

Comment:
We need the last attempt at creating the file (the one that wr thinks succeeded) to be the one we use, but this will make all retries fail.

See https://stackoverflow.com/a/58148921/675083 and do a rename instead; manually test the rename doesn't return an error.

Again, test this for real please and report results.